digital_image_processing/rotation/rotation.py

```diff
@@ -1,11 +1,12 @@
 import cv2
 import numpy as np
 from matplotlib import pyplot as plt
+import os.path
```

Comment:
Please use https://docs.python.org/3/library/pathlib.html instead of os.path for improved readability.

Reply:
Path(__file__).resolve().parent / "image_data" / "lena.jpg"